HW02/Mikhail_Sauchuk/mikhail_sauchuk_hw_02_t_01.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# Method `task_1` should return:
+# - full text of the first line with an error
+# (the presence of the substring `error` in any case)
+# - If the input is text that does not contain the word `error`,
+# return an empty string
+
+LOG_FILE = 'bash.log'
+
+def main
+  puts 'Please enter the full path to log file you would like to chekc or ' \
+    'press Enter to test the code'
+  user_input = gets.chomp
+  log_to_check = check_input(user_input)
+  find_error(log_to_check)
+end
+
+def find_error(file_name)
+  # method rerturns first line with error from the input log
+  lines = File.readlines(file_name)
+  error_lines = lines.find_all { |line| line.downcase.include? 'error' }
```

Comment:
We can use `find` to return only one record
```suggestion
  error_lines = lines.find { |line| line.downcase.include? 'error' }
```

Reply:
Replaced find_all with find method